superdesk/core/utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2024 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
```

Comment:
We probably don't need to keep adding this as the License is in the repo and the link is broken anyways.

Reply:
Good point. We can remove those all later from superdesk.core